# Supplementary data for the manuscript in press, currently accepted in Blood

A comprehensive transcriptome signature of murine hematopoietic stem cell aging

Arthur Flohr Svendsen<sup>1</sup>, Daozheng Yang<sup>1</sup>, KyungMok Kim<sup>2</sup>, Seka Lazare<sup>1</sup>, Erik Zwart<sup>1</sup>, Anna Mura-Meszaro<sup>2</sup>, Albertina Ausema<sup>1</sup>, Bj√∂rn von Eijss<sup>2</sup>, Gerald de Haan<sup>1</sup>, Leonid V. Bystrykh<sup>1</sup> (l.bystrykh@umcg.nl) 

<sup>1</sup>European Research Institute for the Biology of Ageing, University Medical Center Groningen, University of Groningen, Antonius Deusinglaan 1, 9713 AV, Groningen, The Netherlands. <br>
<sup>2</sup>Transcriptional Control of Tissue Homeostasis Lab, Leibniz Institute on Aging, Fritz Lipmann Institute e.V., Beutenbergstr. 11,07745 Jena, Germany

Mostly python scripts by biologist for biologists

Details are in the branches

Branch scripts contain scripts for jupyter or colab (.ipynb format)
There are three branches with source files: meta (for differential expression data as reported by authors), rean (for reanalysed by us data) , AS_lists (lists of differentially expressed genes from the aging signature per publication).
